src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    x: 0,
+    y: 0,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    
+    goRight(right = 1) {
+      if (right > 0) {
+        this.x += right;
```

Comment:
```suggestion
        this.coords.x += right;
```
В об'єкті robot має бути об'єкт coords, в якого є x та y